sympy/functions/combinatorial/factorials.py

```diff
@@ -209,10 +209,15 @@ class subfactorial(CombinatorialFunction):
     It can also be written as int(round(n!/exp(1))) but the recursive
     definition with caching is implemented for this function.
 
-    This function is generalized to noninteger arguments [2]_ as
+    This function is generalized [2]_ as
```

Comment:
I would rather suggest something like:
An interesting analytic expression is the following [2]_